### ISSUE TITLE
Feature Request: Cycle Cameras and set main view size.

### ISSUE THREAD

**Issue by Tovrin** [custom-cards/surveillance-card#27]
As I don't motion control with my cameras, it would be handy just to be able to cycle through the cameras, and set the interval.

The second one, I'm not sure if it's already possible, but the picture size is only very small.  Is it possible to set the size, or have it dynamically sized to take up space available (and maintain aspect ratio, of course)

**Comment by nagyrobi** [comment]
+1 for this

**Comment by ielbury** [comment]
It would be great if it could switch to cycle some time after the last motion.

**Comment by gbdesai2004** [comment]
Yes, I second that.  Cycle a set of cameras ever n time, then when there is motion switch to the camera with motion and follow it from camera to camera till some sort of timeout after motion stops, then back to cycling.

**Comment by niemyjski** [comment]
Good idea, any takers for submitting a pr once #28 is merged in?

**Comment by stingly** [comment]
I have made some changes to my local file to enable automatic transitioning to the next camera in the list, controlled by a new attribute in the lovelace yaml file. I'm quite happy to create a PR for it, but I don't have motion sensors on my cameras so cannot test how that would work with my new code.

I _think_ that it will switch to the motion-detected camera and then move on after n time to the next in the list...

**Comment by stingly** [comment]
I attempted to push my branch to the repository but I guess I need to be allowed to do that as I get this error:
```
ERROR: Permission to custom-cards/surveillance-card.git denied to stingly.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by niemyjski** [comment]
@stingly really sorry I just saw this. Please open a pr for this, you can do so by forking this repo and pushing to your fork. GitHub will then open a pr for this.

**Comment by Pilarmb93** [comment]
> Sí, secundo eso. Realice un ciclo de un conjunto de cámaras cada n tiempo, luego, cuando haya movimiento, cambie a la cámara con movimiento y sígalo de cámara en cámara hasta algún tipo de tiempo de espera después de que el movimiento se detenga, luego vuelva al ciclo.

Does anyone know if this improvement has been made?
It would be very good for me.

**Comment by Pilarmb93** [comment]
> As I don't motion control with my cameras, it would be handy just to be able to cycle through the cameras, and set the interval.
> 
> The second one, I'm not sure if it's already possible, but the picture size is only very small. Is it possible to set the size, or have it dynamically sized to take up space available (and maintain aspect ratio, of course)

Regarding the resizing to fit the available space, is there any progress?

**Comment by niemyjski** [comment]
@Pilarmb93 no, there hasn't been any pr's for this opened.

**Comment by Pilarmb93** [comment]
> @Pilarmb93 no, there hasn't been any pr's for this opened.

What a pity. This is very necessary.
Hopefully @stingly sees it and shares his changes with us.

**Comment by stingly** [comment]
I've pushed up my changes (I think) - PR #59 - My code is probably very out of date now though as I solved this 2 years ago... I've not been using Home Assistant for over a year at this point, so I apologise if the code is unusable

**Comment by Pilarmb93** [comment]
> I've pushed up my changes (I think) - PR #59 - My code is probably very out of date now though as I solved this 2 years ago... I've not been using Home Assistant for over a year at this point, so I apologise if the code is unusable

@niemyjski the PR is already

**Comment by niemyjski** [comment]
@stingly @Pilarmb93 there are merge conflicts. I also feel like this may need a setting to enable it. Can you please work on fixing this @Pilarmb93

**Comment by Pilarmb93** [comment]
@niemyjski
I don't know how to do that, sorry.
I will continue to search for an integration that can offer me what I need.
Thanks.
